public/static/docs/command-reference/repro.md

```diff
@@ -28,6 +28,9 @@ then be executed in the correct order, reproducing pipeline results.
 > Pipeline stages are typically defined using the `dvc run` command, while
 > initial data dependencies can be registered by the `dvc add` command.
 
+💡 For convenience, a Git hook is available to remind you to `dvc repro` when
```

Comment:
it's not necessarily `dvc repro` though. It might be `dvc commit` or `dvc checkout -f` or manually remove files ... it varies - we just run status, that's it

Reply:
so, not even sure it worth having this note here

Reply:
Hmmm well, that's what the `dvc install` ref says it does: https://github.com/iterative/dvc.org/blob/master/public/static/docs/command-reference/install.md

But it sounds like I should try this and confirm that doc first.

Reply:
> The installed Git hook automates reminding the user to run either dvc repro or dvc commit, as needed.

https://dvc.org/doc/command-reference/install

Reply:
that's should be easy enough to fix, it's two functions 5-10 lines of code each - https://github.com/iterative/dvc/blob/master/dvc/scm/git/__init__.py#L277 . The purpose of the initial doc was probably to clarify when it might be useful to see the status and fail (forgot to commit, repro, manually, may be even push (I would check this one), basically every situation status return non-zero exit code and what does it mean from a user perspective).

Reply:
OK. I'm extracting this to a new issue #968